BlueMapSponge/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    shadow "org.spongepowered:spongeapi:7.1.0-SNAPSHOT"
+    compile "org.spongepowered:spongeapi:7.1.0-SNAPSHOT"
```

Comment:
This results in including the whole SpongeAPI and its dependencies in the jar

Reply:
Yeah, if gradle 5 works for you. You can undo some of the downgrades on my pr branch, and merge. Like I stated earlier I did not get the issue on my mac so it may just be a fluke.